client/src/components/SignUp/SignUp.js

```diff
@@ -86,7 +86,7 @@ class FormPage extends React.Component {
               </MDBBtn>
             </div>
           </form>
-          {/* <MDBRow>
```

Comment:
Everything looks good besides this, can you either comment it back out or delete this section, if not there will be two sign up forms

Reply:
Removed it